FLEDGE.md

```diff
@@ -743,6 +743,66 @@ For detailed specification and explainers of the trusted key-value server, see a
 
 ##### 3.1.1 Cross-Origin Trusted Server Signals
 
+If the key-value server is on a different origin than the corresponding script, additional steps are
+needed to ensure that sensitive information is not inappropriately leaked to, or is manipulated by,
+a third party.
+
+In case of bidder signals, the following changes occur:
+1. The fetch is a cross-origin CORS fetch with `Origin:` set to the buyer script's origin.
+2. The value is passed to `crossOriginTrustedSignals` parameter, not the `trustedBiddingSignals`
+   parameter, and there is one more level of nesting denoting the server's origin, e.g:
+
+```
+{
+  'https://www.kv-server.example': {
+    'keys': {
+        'key1': arbitrary_json,
+        'key2': arbitrary_json,
+        ...},
+    'perInterestGroupData': {
+        'name1': {
+            'priorityVector': {
+                'signal1': number,
+                'signal2': number,
+                ...}
+        },
+        ...
+    }
+  }
+}
+```
+
+Seller signals have additional requirements, as the `trustedScoringSignalsURL` is provided by the
+publisher, not the seller:
+1. The seller script must provide an `Ad-Auction-Allow-Trusted-Scoring-Signals-From` response header,
+   a  [structured headers list of strings](https://www.rfc-editor.org/rfc/rfc8941) describing origins
+   fetching trusted signals from which is permitted. The trusted scoring signals fetch may not begin
```

Comment:
describing origins from which fetching trusted signals is permitted

Reply:
Applied, thanks